mafiasi/gprot/templates/gprot/index.html

```diff
@@ -52,6 +54,18 @@
         </td>
         <td>{{ gprot.exam_date|date:"SHORT_DATE_FORMAT" }}</td>
         <td>{{ gprot.examiners|format_examiners }}</td>
+        <td>
+            {% for label in gprot.labels.all %}
+            <span class="label label-{{ label.colour }}">{{ label.tag }}</span>
+            {% endfor %}
+        </td>
+        <td style="text-align: center">
+            {% if gprot.is_pdf %}
+            <img src="{% static "img/icon-yes.svg" %}"/>
```

Comment:
Does Bootstrap not have such icons already? Is there a need to use image files for it?